backend/app/tests/services/llm/test_guardrails.py

```
@@ -253,3 +267,108 @@ def test_list_validators_config_network_error_fails_open(mock_client_cls) -> Non
+        kwargs = dict(
+            session=db,
+            query=QueryParams(input="original unsafe input"),
+            config=_CONFIG,
+            request_metadata=None,
+            config_blob=_CONFIG_BLOB,
+            guardrail_direct_response=_SAFE_TEXT,
+            job_id=job.id,
+            project_id=project.id,
+            organization_id=project.organization_id,
+            chain_id=None,
+        )
```

Ruff (0.15.12)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)